controllers/placementrule/manifestwork.go

```diff
@@ -312,14 +311,26 @@ func getAllowList(client client.Client, name string) (*MetricsAllowlist, error)
 	return allowlist, nil
 }
 
-func handleDeletedMetrics(metricNameList []string) []string {
-	nameList := []string{}
-	for _, name := range metricNameList {
+func handleDeletedMetrics(defaultAllowlist []string, customAllowlist []string) []string {
```

Comment:
Suggest to use `mergeMetrics` as function name, because it is for both `append` and `delete`.

Reply:
Thank you for your advice.